lib/cfxxl.ex

```diff
@@ -161,6 +164,7 @@ defmodule CFXXL do
 
   def certinfo(_client, _opts), do: {:error, :no_certificate_or_domain}
 
+  @spec crl(CFXXL.Client.t(), any) :: {:ok, any} | {:error, any}
```

Comment:
Try to be specific with input arguments when you can deduce more specific types from the contex and/or from the doc. For example here you could use:
```
   @spec crl(CFXXL.Client.t(), String.t() | nil) :: {:ok, any} | {:error, any}
```